src/navigation/index.tsx

```diff
@@ -10,6 +10,7 @@ import { Content } from "./Content";
 import { Providers } from "../context/composeProviders";
 import { DrawerContextProvider } from "../context/drawer";
 import { CampaignConfigsStoreContextProvider } from "../context/campaignConfigsStore";
+import "../common/i18n/i18nSetup";
```

Comment:
Why do we need to import this?

Reply:
The i18n setup was abstracted and needs to be imported for the initialisation! Without it, will not work